### ISSUE TITLE
As a Gitcoin Admin, I'd like the issues I've re-marketed to surface to the top of the issue explorer as a newly opened issue.

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#3022]
**To be worked on after #2184 is completed**

### User Story

As a Gitcoin Admin, I'd like the issues I've re-marketed to surface to the top of the issue explorer as a newly opened issue.

As a Contributor, I'd like to see re-marketed issues at the top of the issue explorer so I'll know that it is open and available to be worked on.

### What 
When an issue is remarketed, it should surface to the top of the issue explorer and show up as a new issue in notifiers-gitcoin on slack.

### Why
This will help us move issues through the platform.

### Definition of Done

- [ ] [Reuse this functionality](https://github.com/gitcoinco/web/issues/2184) for surfacing to the top of the issue explorer that has been built for Expired Issues.
- [ ] Post a video or screenshots of the feature built
- [ ] This issue passes all of the tests and revisions from the Gitcoin Core engineering team.

**Comment by owocki** [comment]
@PixelantDesign as were building the logic for this, i'd like us to also consider building a premium feature wheren ppl can pay to make their bounty 'promoted'.   i can write up a seperate ticket if need

**Comment by PixelantDesign** [comment]
@owocki we have something like this in the back log #2390  and build ticket #2720. Open to other ideas though.
